### PR TITLE
(MAINT) Bump puppet and puppet-agent versions 26-apr-2017

### PR DESCRIPTION
**Pull request by adrienthebo**

### CODE REVIEW

File: acceptance/lib/helper.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "b04c9f4a8d4eace424a718073315a7da0a34f3b5",
+                         "2aa05621ebb25ce724760ab4e65425b8a55f3f60",
```
Comment (adrienthebo) [inline acceptance/lib/helper.rb]: This is supposed to be the puppet-agent commit matching `1.10.0.337.g2aa0562` right?

**Comment by camlow325** [comment]
Looks like we had a rubygems / network failure related to the spec tests on one of the Travis cells:

```
Connection reset by peer (https://api.rubygems.org/gems/bundler-1.14.6.gem)
```

I'll reload that and see if it recurs.

File: acceptance/lib/helper.rb (hunk 1 above)
Comment (camlow325) [inline acceptance/lib/helper.rb]: Yep, that's what we've been doing.  Looks right to me.

**Comment by rlinehan** [comment]
Looks like travis passed? That seems fast. But I'm 👍

**Pull request merged**